src/Notification/Notification.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
```

Comment:
Resolve PR comment about how elements on the screenshot differs from the mockups

Reply:
Default notification
![image](https://user-images.githubusercontent.com/47984484/53829532-5da92d00-3fa6-11e9-9485-d516a170c38e.png)

Notification with solid fill colors
![image](https://user-images.githubusercontent.com/47984484/53829571-6e59a300-3fa6-11e9-879c-cb119d12ff69.png)

Notification with actions
![image](https://user-images.githubusercontent.com/47984484/53829450-33576f80-3fa6-11e9-8560-2becd6b3fa73.png)
![image](https://user-images.githubusercontent.com/47984484/53829492-48cc9980-3fa6-11e9-8f66-174a7498efa7.png)

@DmitriVanGuard have a look at the updated screenshots please